docs/docs/reference/metaprogramming/macros-spec.md

```diff
@@ -196,6 +196,8 @@ private def dynamicPower(x: Double, n: Int): Double =
   else x * dynamicPower(x, n - 1)
 ```
 
+This assumes a `.value` that maps tree nodes representing constants to their values.
+
```

Comment:
This seems to be not necessary, as `.value` indeed exists in stdlib.

Reply:
It comes from @nicolasstucki's suggestion (see [his comment](https://github.com/lampepfl/dotty/pull/10826#discussion_r545216639) in PR #10826).

Reply:
Maybe change it to something like the following:

> In the above, the method `.value` maps expressions representing constants of the type `Expr[T]` to their values of the type `T`.

Reply:
Done.